diff --git a/pam.d/common-session-noninteractive b/pam.d/common-session-noninteractive
index 1fee2c4..7d3c962 100644
--- a/pam.d/common-session-noninteractive
+++ b/pam.d/common-session-noninteractive
@@ -27,4 +27,5 @@ session	required			pam_permit.so
 session optional			pam_umask.so
 # and here are more per-package modules (the "Additional" block)
 session	required	pam_unix.so 
+session required	pam_limits.so
 # end of pam-auth-update config

